src/components/equitySection/equitySection.js

```diff
@@ -6,23 +6,23 @@ import { getImage } from "gatsby-plugin-image";
 import { BgImage } from "gbimage-bridge";
 import useQuoteImages from "../../hooks/useQuoteImages";
 
-import content from "../../pages/equity.content.yml";
+import indexContent from "../../pages/index.content.yml";
+import stateContent from "../../pages/state/[state].content.yml";
 
 import * as styles from "./equitySection.module.css";
 
-export default function EquitySection({ size = "lg" }) {
+export default function EquitySection({ location, size = "lg" }) {
```

Comment:
This is my b. It's not appropriately descriptive, but we're already using `size` for what we're doing here, really. `lg` = Main Page, `sm` = State Page (and defaulting to `lg`/Main)

Reply:
We could change it to something like `page` or `isMainPage`, maybe?

Reply:
I went with page.